test/unit/org/apache/cassandra/metrics/TableMetricsTest.java

```diff
@@ -296,6 +304,22 @@ public void testCounterStatement()
         assertGreaterThan(cfs.metric.coordinatorWriteLatency.getMeanRate(), 0);
     }
 
+    @Test
+    public void testCreateLikeTableMetric()
```

Comment:
@Maxwell-Guo I am not completely sure what is the added value of this test. We can not just test absolutely everything again just because a table was copied. Why is this test necessary? If we copy a table, it is as any other, then it means that it will also behave like any other and we do need to test it for the second time.

You just need to test that after you copy a table, that table is equal to the source once, structurally, and is independent from the original one (e.g. modification / alterations of fields and changing options does not change the options on the original table).